recipes/r-kmsurv/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set version = '0.1-5' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-kmsurv
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: KMsurv_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/KMsurv_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/KMsurv/KMsurv_{{ version }}.tar.gz
+  sha256: 55627bc301245e0a4d6be04565fbaabd9cc360e2e5e4bc1e38341354e6de2542
+
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+
+  run:
+    - r-base
+
+test:
+  commands:
+    - $R -e "library('KMsurv')"  # [not win]
+    - "\"%R%\" -e \"library('KMsurv')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=KMsurv
+  license: GPL (>= 3)
```

Comment:
GPL requires that the license be distributed with the package. Please include a `license_file` entry.